docs/cli/example_config.toml

```diff
@@ -61,7 +61,7 @@ devfakeauthor = false           # Run miner without validator set authorization
   nolocals = false              # Disables price exemptions for locally submitted transactions
   journal = "transactions.rlp"  # Disk journal for local transaction to survive node restarts
   rejournal = "1h0m0s"          # Time interval to regenerate the local transaction journal
-  pricelimit = 30000000000      # Minimum gas price limit to enforce for acceptance into the pool. Recommended for mainnet = 30000000000 (not enforced). For Amoy network, regardless the value set, it will be enforced to 25000000000 in bor.
+  pricelimit = 30000000000      # Minimum gas price limit to enforce for acceptance into the pool. Regardless the value set, it will be enforced to 25000000000 for all networks
```

Comment:
If it is anyhow set to `25000000000` can we change the value here?

Reply:
But this is good as they would know this is not being honoured and it's being enforced in the client codebase itself.

Reply:
Yeah sorry my bad. Forgot to change here.